src/ProgramBlockEditor.js

```diff
@@ -82,13 +86,22 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, {}> {
     };
 
     makeProgramBlock(programStepNumber: number, command: string) {
+        const active = this.props.activeProgramStepNum === programStepNumber;
+        let classNames = [
+            'ProgramBlockEditor__program-block',
+            'command-block'
+        ];
+        if (active) {
+            classNames.push('ProgramBlockEditor__program-block--active');
+        }
         switch(command) {
             case 'forward':
                 return (
                     <Button
+                        ref={active ? r => (this.commandBlock = r) : null}
```

Comment:
Please move the logic from this anonymous function to a method on ProgramBlockEditor: `setActiveCommandBlockRef` and then the JSX becomes:

```
ref={this.setActiveCommandBlockRef}
```

For an example, see: https://reactjs.org/docs/refs-and-the-dom.html#callback-refs